databases/tests/test_errors.py

```diff
@@ -39,19 +39,19 @@ async def test_field_not_found_error(client: Prisma) -> None:
         )
 
 
-@pytest.mark.asyncio
-@pytest.mark.prisma
-async def test_field_not_found_error_selection() -> None:
-    """The FieldNotFoundError is raised when an unknown field is passed to selections."""
+# @pytest.mark.asyncio
+# @pytest.mark.prisma
+# async def test_field_not_found_error_selection() -> None:
+#     """The FieldNotFoundError is raised when an unknown field is passed to selections."""
 
-    class CustomPost(prisma.bases.BasePost):
-        foo_field: str
+#     class CustomPost(prisma.bases.BasePost):
+#         foo_field: str
 
-    with pytest.raises(
-        FieldNotFoundError,
-        match=r'Field \'foo_field\' not found in enclosing type \'Post\'',
-    ):
-        await CustomPost.prisma().find_first()
+#     with pytest.raises(
+#         FieldNotFoundError,
+#         match=r'Field \'foo_field\' not found in enclosing type \'Post\'',
+#     ):
+#         await CustomPost.prisma().find_first()
```

Comment:
> [!CAUTION]
> `test_field_not_found_error_selection` skipped

JSON protocol does not explicitly select all fields, so it won't raise a `FieldNotFoundError` here.

However we could:
- explicitly select all fields (cons: this could be hard to impl and bring overhead to serializer, prisma-engine and maybe even db); or...
- leave it to pydantic as it will raise validation errors if data doesn't match the model.